sdk/python/feast/infra/feature_servers/multicloud/Dockerfile

```diff
@@ -7,8 +7,7 @@ RUN apt update && \
         build-essential
 
 RUN pip install pip --upgrade
-RUN pip install "feast[aws,gcp,snowflake,redis,go,mysql,postgres]"
-
+RUN pip install "feast[aws,gcp,snowflake,redis,go,mysql,postgres,opentelemetry]"
```

Comment:
as previously discussed, the Dockerfile.dev file should also be modified accordingly